index.bs

```diff
@@ -234,9 +234,14 @@ interface Operand {};
 <script type=idl>
 typedef double number;
 
+dictionary Output {
+  required DOMString name;
+  required Operand operand;
+};
+
```

Comment:
Good point.

Offering one perspective to help you find good names for these things:

Since dictionary identifier name (here Output) is not surfaced to web developers the name can be more verbose if it makes the spec more clear.

Proper naming of dictionary members (here name and operand) is important however, since they are exposed and stick around.

Reply:
Fixed in the new commit. Please take another look.